model_api/python/model_api/tilers/detection.py

```diff
@@ -123,8 +123,9 @@ def _merge_results(self, results, shape):
                 iou_threshold=self.iou_threshold,
             )
 
+        start_idx = 1 if self.tile_with_full_img else 0
         merged_vector = (
-            np.mean(feature_vectors, axis=0) if feature_vectors else np.ndarray(0)
+            np.mean(feature_vectors[start_idx:], axis=0) if feature_vectors else np.ndarray(0)
```

Comment:
Why do we need to change the logic for feature_vector?

Reply:
I have aligned this with the PyTorch implementation: https://github.com/openvinotoolkit/training_extensions/pull/3530/files#diff-3e70350ee460c59caf89aff057b556900998828cf13f7a1ebe802e89068fd6e1R210.

I'm unsure which approach provides the best representation. @GalyaZalesskaya, could you share your opinion?

Reply:
Yes, basically for feature vector it doesn't matter whether we use first tile with the whole image or not. I thought we can align approach for saliency maps (where we use tile with image in a special way) and skip the tile with image for feature vector.

But now it seems to me that for feature vector it's better to leave the initial implementation in PyTorch and OpenVINO.
@eugene123tw Can I ask you to revert changes?

Reply:
reverted